app/src/modules/settings/routes/data-model/field-detail/components/relationship-m2a.vue

```diff
@@ -191,6 +191,10 @@
 			<v-icon class="arrow" name="arrow_backward" />
 			<v-icon class="arrow" name="arrow_backward" />
 		</div>
+
+		<v-notice v-if="generationInfo !== null" type="warning">
+			<span v-html="generationInfo"></span>
```

Comment:
Could we do this without `v-html`? Something like:

```suggestion
			<span>{{ $t('new_data_alert') }}</span>
			<ul>
				<li v-for="newItem in newItems">Etc</li>
			</ul>
```

Reply:
Maybe but why?

Reply:
Because it's cleaner then generating the HTML in JS and then injecting it in unsafe v-html